src/rt/sched/schedulerthread.h

```diff
@@ -449,8 +594,13 @@ namespace verona::rt
           UNUSED(tsc);
         }
 #endif
-          // Enter sleep only when the queue doesn't contain any real cowns.
-          if (state == ThreadState::NotInLD && q.is_empty())
+          if ((mute_map.size() != 0) || (unmute_q.peek() != nullptr))
```

Comment:
Formatting looks off here, but I couldn't figure it out well enough to do a suggested change.